incident/models/incident_page.py

```diff
@@ -798,7 +798,11 @@ def get_court_circuit(self):
         return None
 
     def get_meta_image(self):
-        return self.teaser_image or super(IncidentPage, self).get_meta_image()
+        return (
+            self.teaser_image or
+            self.get_main_category().default_image or
+            super(IncidentPage, self).get_meta_image()
```

Comment:
We might not be able to get away with calling `super().get_meta_image` here, since what that function does is either return the incident's `search_image` field _or_ the site-wide default image. So if we only call this after we check for the category default image, the incident-level search image will not be shown in any categories with a default image.

Though it's possible we are not using the `search_image` field, in which case maybe it does not matter so much.